sdk/digitaltwins/Azure.DigitalTwins.Core/tests/AdtQueryBuilderTests/AdtQueryBuilderTests.cs

```diff
@@ -35,7 +35,7 @@ public void SelectSinglePropertyTests()
     }
 
         [Test]
-        public void SelectMultiplePropertyTests()
+        public void AdtQueryBuilder_Select_MultipleProperties_Two()
```

Comment:
I would even go as far as removing this test case and renaming the test case below to omit the count "Four". Up to you